src/commonMain/kotlin/exchange.dydx.abacus/output/input/TradeInput.kt

```diff
@@ -709,7 +709,8 @@ enum class OrderStatus(val rawValue: String) {
     `open`("OPEN"),
     pending("PENDING"),
     untriggered("UNTRIGGERED"),
-    partiallyFilled("PARTIALLY_FILLED");
+    partiallyFilled("PARTIALLY_FILLED"),
+    partiallyCanceled("PARTIALLY_CANCELED");
```

Comment:
can we add a comment about the differences here? (basically your PR description)